Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -940,8 +940,10 @@ static void webkitWebViewConstructed(GObject* object)
 #elif PLATFORM(WPE) && ENABLE(WPE_PLATFORM)
     if (priv->display) {
         GRefPtr<WebKitInputMethodContext> imContext = adoptGRef(webkitInputMethodContextImplWPENew(priv->view->wpeView()));
-        webkitInputMethodContextSetWebView(imContext.get(), webView);
-        priv->view->setInputMethodContext(imContext.get());
+        if (imContext) {
+            webkitInputMethodContextSetWebView(imContext.get(), webView);
+            priv->view->setInputMethodContext(imContext.get());
+        }
```

Comment:
Oh, I didn't realize this was possible when reviewed the IME patch. This should be fixed in a different patch. We should add a None implementation as a fallback for platforms not supporting/implementing IME.

Reply:
Thanks. Already checked https://commits.webkit.org/280009@main and indeed I'm getting no crashes regarding IM.